tests/projects/schema.py

```diff
@@ -282,13 +296,17 @@ class TagInputPartial(NodeInputPartial):
 @strawberry_django.input(Issue)
 class IssueInput:
     name: strawberry.auto
-    milestone: "MilestoneInputPartial"
     priority: strawberry.auto
     kind: strawberry.auto
     tags: Optional[List[NodeInput]]
     extra: Optional[str] = strawberry.field(default=UNSET, graphql_type=Optional[int])
 
 
+@strawberry_django.input(Issue)
+class IssueInputWithMilestones(IssueInput):
+    milestone: "MilestoneInputPartial"
+
```

Comment:
Workaround not to expose milestone related fields to inherited schema (as related input types were updated to cover nested CUD operations): https://github.com/strawberry-graphql/strawberry-django/blob/9025ee5a0af7c31ce8fffe2be9ce9d871b289a17/tests/projects/test_schema.py#L21-L34